include/convex_bodies/orderpolytope.h

```diff
@@ -0,0 +1,482 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+// Copyright (c) 2020-2021 Vaibhav Thakkar
+
+// Contributed and/or modified by Vaibhav Thakkar, as part of Google Summer of Code 2021 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef ORDER_POLYTOPE_H
+#define ORDER_POLYTOPE_H
+
+#include <iostream>
+#include "poset.h"
+#include <Eigen/Eigen>
+#include "preprocess/max_inscribed_ball.hpp"
+#ifndef VOLESTIPY
+    #include "lp_oracles/solve_lp.h"
+#endif
+
+
+template <typename Point>
```

Comment:
does it make sense to inherit Hpolytope and then implement only the functions that are different for the order polytope?

Reply:
this can be done, but majority of the functions of the OrderPolytope class have different implementations compared to the HPolytope class.